requirements/requirements-py-3.12.txt

```diff
@@ -148,6 +148,7 @@ pytest-datadir==1.5.0
 pytest-regressions==2.5.0
 pytest-rerunfailures==12.0
 pytest-timeout==2.2.0
+pytest-xdist==3.6.1
```

Comment:
This is exactly why we should use a universal lock file :-)

Reply:
I feel the same.